src/commcare_cloud/commands/migrations/couchdb.py

```diff
@@ -53,11 +53,22 @@ class MigrateCouchdb(CommandBase):
             - commit: update database docs with new shard allocation
             - clean: remove shard files from hosts where they aren't needed
         """),
+        Argument('--no-stop', action='store_true', help="""
+            When used with migrate, operate on live couchdb cluster without stopping nodes
```

Comment:
Is there any danger to doing this? Like will there be any data loss if the cluster is in use during the migration?

Reply:
I think if you're moving shards so that the set of new nodes and old nodes is disjoint then yes there's danger of losing data. But if you are moving it so that for every shard there's a place that it was both before and after, then whatever bit is missed on the new shards will get replicated over from the "stable pillar" shard.

Or, as in my case, if you're doing this on a database that isn't getting written to. I had stopped the replication, and there was no live process writing to this db, so it was just a matter of transferring static data.

Reply:
I think it's worth making a note of some of these points in the docs somewhere.

Reply:
:+1: